synapse/models/infotech.py

```diff
@@ -934,6 +956,103 @@ def getModelDefs(self):
                         'doc': 'An array of ChildOf CWE Relationships.'
                     }),
                 )),
+
+                ('it:sec:metrics', {}, (
+
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization whose security program is being measured.'}),
+
+                    ('org:name', ('ou:name', {}), {
+                        'doc': 'The organization name. Used for entity resolution.'}),
+
+                    ('org:fqdn', ('inet:fqdn', {}), {
+                        'doc': 'The organization FQDN. Used for entity resolution.'}),
+
+                    ('window', ('ival', {}), {
```

Comment:
Did we decide on window or timeframe for the de-facto "time window" (interval) property?

Reply:
went with `period` :+1: